src/cli.js

```diff
@@ -7,4 +7,5 @@ commander
     .allowUnknownOption()
     .command('apply', 'Apply config to a kong server', {isDefault: true})
     .command('dump', 'Dump the configuration from a kong server')
+    .command('prune', 'Remove any items marked "ensure: removed"')
```

Comment:
I could see this being read as 'Remove any items marked "ensure: removed" _from the server_'. Maybe specify `from your config file`?

Reply:
How about a more positive, active phrasing like _"Rewrites config, omitting any items marked "ensure: removed""_